packages/react-i18n/src/i18n.ts

```diff
@@ -1,3 +1,6 @@
+import {memoize as memoizeFn} from '@shopify/function-enhancers';
+import {memoize} from '@shopify/decorators';
+import {languageFromLocale, regionFromLocale} from '@shopify/i18n';
```

Comment:
With the resolver changes and re-classification of these libraries (from "internal" to "external"), they are being rearranged as a result of `import/order`.